pkg/virt-launcher/monitor_test.go

```diff
@@ -81,8 +95,9 @@ var _ = Describe("VirtLauncher", func() {
 	}
 
 	BeforeEach(func() {
+		os.Remove(pidFile)
```

Comment:
you're right, it doesn't get cleaned up. That wasn't intentional. 

I'm going to try working on a solution that does pid detection without a pidfile.  Keeping up with that pid file is annoying.

Reply:
I kind of like the pidfile. It allows the container to see, if we tried at start before. Otherwise we can miss that, when we do cgroup scanning.

Reply:
> I kind of like the pidfile. It allows the container to see, if we tried at start before. Otherwise we can miss that, when we do cgroup scanning.

until we make the UUID change, I'm hesitant to infer/couple any logic with the presence of a pidfile other than communicating the pid.

For now, I went ahead an removed the pidfile from this patch. I think it simplifies everything even further. Independently of pid detection, we can still communicate with virt-launcher that the the VM attempted to start using a file later on if we want.